### ISSUE TITLE
Better support for actions in Group Chat

### ISSUE THREAD

**Issue by owliwar** [nlef/moonraker-telegram-bot#308]
### What would you like to see added?

I have 3 printers in my home that I added in a group chat with my Partner, so they can also receive notifications and stop printing if something goes wrong and I'm busy attending other stuff. it has happened before more than once before, so Its a usefull thing to have in an emergency. I was using octoprint telegram bot in that time. 

Right now, my workaround is adding this groupchat id to the chatID, but the way the bot handles it is not perfect. the buttons are merged, and they both respond, but only if they were the last one to send a notification or if I @them with the action, like  /status@botname (no spaces) 

I'm not sure if the actions on the chat are overwriten by the bot, but when I was using octoprint telegram bot they could co-exist in the same action menu (with @printername/action) 

TLDR 
I'd like to have at least /status and /pause or /stop print for each bot in a group chat

**Comment by nlef** [comment]
Hi,
Looks like it's a dulicate of #283 
Any development in this direction is possible only after the bot is transferred to asincio to support new versions of python-telegram-bot library.

**Comment by owliwar** [comment]
oh, thanks!

**Comment by aka13-404** [comment]
Closing this as main discussion is in #283
